hazelcast/src/main/java/com/hazelcast/internal/eventlog/EventLogService.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (c) 2008-2019, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.internal.eventlog;
+
+/**
+ * Event log Service.
+ */
+public interface EventLogService {
```

Comment:
I'd consider renaming this service to `AuditLogService` or similar since this is meant not to be a general-purpose event logging system (as I understand). The same for the package name and the `NodeExtension`'s getter.

Reply:
Covered your points and rebased. Could you have a look, please?